packages/shr-fhir-export/lib/extensions.js

```diff
@@ -567,32 +567,35 @@ class ExtensionExporter {
         extension.snapshot.element = extension.snapshot.element.filter(el => el.id !== newId);
       }
       // This avoids inlining the uneccessary top level "Extension" element or readding value that was constrained
-      if (splitPoint < 0 || (existingElement != null && pathEnd.startsWith('value'))) {
-        continue;
-      } else {
```

Comment:
I realize this was refactored based on @mint-thompson's comments, but it seems to me that the end result is different now.

Before, if `splitPoint < 0 || (existingElement != null && pathEnd.startsWith('value'))`, we went immediately to the next iteration of the loop.  Now we would fall through the new inverted `if` statement and pick up at line 578 (instead of going to the next iteration).  Is that intentional?

Reply:
Wait.  I might be miscounting my `{` and `}`.  Let me look at that more closely.

Reply:
NEVERMIND.  I did miscount the `}`.